### ISSUE TITLE
[Bug] Build failed Ubuntu 18.04

### ISSUE THREAD

**Issue by blockchain-trainer** [libra/libra#212]
# 🐛 Bug

./start_cli_testnet.sh  failed the build

## To reproduce

** Code snippet to reproduce **
./start_cli_testnet.sh 

# Your code goes here
# Please make sure it does not require any external dependencies
```

** Stack trace/error message **
```
Compiling noise v0.1.0 (/media/montoo/fc51dc5b-56d1-4d01-8310-d7b0048238d5/libra/libra/network/noise)
error: failed to run custom build command for `mempool v0.1.0 (/media/montoo/fc51dc5b-56d1-4d01-8310-d7b0048238d5/libra/libra/mempool)`

Caused by:
  process didn't exit successfully: `/media/montoo/fc51dc5b-56d1-4d01-8310-d7b0048238d5/libra/libra/target/debug/build/mempool-50d552cb4604c146/build-script-build` (exit code: 101)
--- stdout
cargo:rerun-if-changed=src/proto/shared/mempool_status.proto
cargo:rerun-if-changed=../types/src/proto/vm_errors.proto
cargo:rerun-if-changed=../types/src/proto/validator_change.proto
cargo:rerun-if-changed=../types/src/proto/get_with_proof.proto
cargo:rerun-if-changed=../types/src/proto/validator_set.proto
cargo:rerun-if-changed=../types/src/proto/account_state_blob.proto
cargo:rerun-if-changed=../types/src/proto/transaction_info.proto
cargo:rerun-if-changed=../types/src/proto/proof.proto
cargo:rerun-if-changed=../types/src/proto/events.proto
cargo:rerun-if-changed=../types/src/proto/access_path.proto
cargo:rerun-if-changed=../types/src/proto/validator_public_keys.proto
cargo:rerun-if-changed=../types/src/proto/transaction.proto
cargo:rerun-if-changed=../types/src/proto/ledger_info.proto
cargo:rerun-if-changed=src/proto/shared/mempool_status.proto
cargo:rerun-if-changed=src/proto/shared/mempool_status.proto

--- stderr
mempool_status.proto: No such file or directory
thread 'main' panicked at 'failed to write descriptor set: Custom { kind: Other, error: StringError("protoc (\"protoc\" \"-I../types/src/proto\" \"-Isrc/proto/shared\" \"-Isrc/proto/shared\" \"--descriptor_set_out=/tmp/871b671a00544986a067bf91861cc1ec\" \"--include_imports\" \"mempool_status.proto\") exited with non-zero exit code") }', src/libcore/result.rs:999:5
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.

warning: build failed, waiting for other jobs to finish...
error: failed to run custom build command for `admission_control_proto v0.1.0 (/media/montoo/fc51dc5b-56d1-4d01-8310-d7b0048238d5/libra/libra/admission_control/admission_control_proto)`

Caused by:
  process didn't exit successfully: `/media/montoo/fc51dc5b-56d1-4d01-8310-d7b0048238d5/libra/libra/target/debug/build/admission_control_proto-b63c4c9f460d2223/build-script-build` (exit code: 101)
--- stdout
cargo:rerun-if-changed=../../types/src/proto/vm_errors.proto
cargo:rerun-if-changed=../../types/src/proto/validator_change.proto
cargo:rerun-if-changed=../../types/src/proto/get_with_proof.proto
cargo:rerun-if-changed=../../types/src/proto/validator_set.proto
cargo:rerun-if-changed=../../types/src/proto/account_state_blob.proto
cargo:rerun-if-changed=../../types/src/proto/transaction_info.proto
cargo:rerun-if-changed=../../types/src/proto/proof.proto
cargo:rerun-if-changed=../../types/src/proto/events.proto
cargo:rerun-if-changed=../../types/src/proto/access_path.proto
cargo:rerun-if-changed=../../types/src/proto/validator_public_keys.proto
cargo:rerun-if-changed=../../types/src/proto/transaction.proto
cargo:rerun-if-changed=../../types/src/proto/ledger_info.proto
cargo:rerun-if-changed=../../mempool/src/proto/shared/mempool_status.proto
cargo:rerun-if-changed=src/proto/admission_control.proto

--- stderr
admission_control.proto: No such file or directory
thread 'main' panicked at 'failed to write descriptor set: Custom { kind: Other, error: StringError("protoc (\"protoc\" \"-I../../types/src/proto\" \"-I../../mempool/src/proto/shared\" \"-Isrc/proto\" \"--descriptor_set_out=/tmp/7d287889e27a4a67a1240379d19865b9\" \"--include_imports\" \"admission_control.proto\") exited with non-zero exit code") }', src/libcore/result.rs:999:5
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.

warning: build failed, waiting for other jobs to finish...
error: build failed

```

## Expected Behavior

<!-- A clear and concise description of what you expected to happen. -->

## System information

**Please complete the following information:**
- <!-- Latest -->
- <!-- downloaded from dev_setup.sh  -->
- <!-- Ubuntu 18.04 -->


## Additional context

Add any other context about the problem here.

**Comment by icarter09** [comment]
@blockchain-trainer I've been going over the all the logged issues and noticed several times that user's are having problems with Ubuntu 18.04. Here are some of the issues: #19, #30, #94, #79. Look them over and see if any of them contain the solution to your current problem.

Also please let me know if one of those fixed the issue as I'm looking to create a section on the README that contains helpful guidance for Ubuntu installs (since this keeps showing up). Thanks.

**Comment by blockchain-trainer** [comment]
I think it was the protoc version, the following solved it in #94 

https://github.com/libra/libra/issues/94#issuecomment-503501334

I think most of the ubuntu machines have some conflicting versions of dependencies, a readme with version checks will help.

thanks
